### ISSUE TITLE
Training

### ISSUE THREAD

**Issue by htaufik** [AlfaCodeFlow/Vehicle_Detection-And-Classification#6]
do you use dataset training for the classification? because when i try with different sample videos, there are some wrong classification

**Comment by AlfaCodeFlow** [comment]
no,i didn't use any data set .Basically it is not a machine learning program. it is simply a computer vision approach .I have  used only image processing tools to create this program. It  can work  with other video properly if there is no shadow issue in that video. Removing shadow is major challange in computer vision.Thats why we use machine learning
